config/500-webhook.yaml

```diff
@@ -32,34 +32,21 @@ spec:
         role: webhook
         events.cloud.google.com/release: devel
     spec:
-      serviceAccountName: controller
+      serviceAccountName: webhook
       containers:
         - name: webhook
+          terminationMessagePolicy: FallbackToLogsOnError
           # This is the Go import path for the binary that is containerized
           # and substituted here.
           image: github.com/google/knative-gcp/cmd/webhook
-          ports:
-          - name: metrics
-            containerPort: 9090
-          imagePullPolicy: Always
-          resources:
```

Comment:
Seems like we should set requests and limits on the webhook pods so they don't get default values.

Reply:
Copied it from eventing. I think these were just numbers copied from serving, due to their e2e tests. See [here](https://github.com/knative/serving/blob/e91fd663c77348557dcfa8720f2a78c167f6d5b9/config/webhook.yaml#L50). 
I'll add them back.